terragrunt/s3.tf

```
@@ -0,0 +1,18 @@
+module "cds_salesforce_backups_bucket" {
+  source = "github.com/cds-snc/terraform-modules?ref=v5.1.8//S3"
+
+  bucket_name        = "cds-salesforce-backups"
+  billing_tag_value  = var.billing_code
+  critical_tag_value = "true"
+  versioning = {
+    "enabled" = true
+  }
+}
```

You'll need a `logging` block to hook this up with the log bucket:
```suggestion
  logging = {
    target_bucket = module.cds_salesforce_backups_logs_bucket.s3_bucket_id
  }
}
```